File: Go4Lunch/app/src/main/java/com/openclassrooms/go4lunch/data/model/api/places/details/OpeningHours.java
package com.openclassrooms.go4lunch.data.model.api.places.details;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Opening Hours Class from Google Place Details Api
 */
public class OpeningHours {

    @SerializedName("weekday_text")
    @Expose
    private List<String> weekdayText = null;

    public OpeningHours() {
    }

    public List<String> getWeekdayText() {
        return weekdayText;
    }
}

File: RealEstateManager/app/src/main/java/com/openclassrooms/realestatemanager/models/place/detail/OpeningHours.kt
package com.openclassrooms.realestatemanager.models.place.detail

import com.google.gson.annotations.SerializedName

data class OpeningHours (
    @SerializedName(value = "open_now")
    val openNow: Boolean,

    val periods: List<Period>,

    @SerializedName(value = "weekday_text")
    val weekdayText: List<String>
)